### ISSUE TITLE
Audio Input empty in iOS Safari 13.7 / Angular 9

### ISSUE THREAD

**Issue by KleptoKat** [twilio/twilio-video.js#1175]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii) or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Expected behavior:**

I'm trying to get Twilio Video setup with the iOS 13.7 on iPad, but my audio input is empty, but not muted. I hooked up an audio indicator and it is at zero, where I expect it to be fluctuating (like it does on other operating systems). The readyState of the mediaStreamTrack is live, enabled is true, muted is false. It works on all other browsers. I've also fixed zone.js in polyfills.ts

I am only calling Video.createLocalTracks once. getUserMedia (from @twilio/rtc) is only called the one time.

Here is a room sid that you can check: RMa79c4909aa7a3ccc20a719142f74de21

This issue is similar, but it looks like it was an issue with his library - however, they did not find a solution: https://github.com/twilio/twilio-video.js/issues/1104 - there was a mention of a fix in 2.7.1, but I am on 2.7.2 and the fix is not working for me.

Am I missing something? Do I need to attach the local audio's HTMLMediaElement to the DOM as well? Is this a bug?

I am running the application using ngrok over HTTPS so my other devices can access it without having to configure certificates for them.

Video and data works perfectly fine.

**Code to reproduce the issue:**

This code is called ONLY ONCE, I've and merged them from separate functions here:

room.service.ts, initLocalTracks() and connect()
```typescript
    const devices = await navigator.mediaDevices.enumerateDevices();
    let videoDeviceId: string;
    let audioDeviceId: string;

    for (const device of devices) {
      if (device.kind === 'audioinput') {
        audioDeviceId = device.deviceId;
      }
      if (device.kind === 'videoinput') {
        videoDeviceId = device.deviceId;
      }
    }

    const tracks: Video.LocalTrack[] = await Video.createLocalTracks({
      logLevel: 'debug',
      video: {
        name: 'camera',
        deviceId: videoDeviceId,
        width: 640,
        frameRate: 20,
        facingMode,
      },
      audio: {

        name: 'microphone',
        deviceId: audioDeviceId,
        // echoCancellation: true,
        // noiseSuppression: false,
      }
    }).catch((reason) => {
      console.warn('Could not create audio track: ', reason);
      // TODO: notify user, open ticket
      return undefined;
    });


    const audioTrack = tracks.find((track) => track.name === MICROPHONE) as Video.LocalAudioTrack;
    const videoTrack = tracks.find((track) => track.name === CAMERA) as Video.LocalVideoTrack;

    if (audioTrack) {
      this.tracks.push(audioTrack);
      this.noMic = false;
    } else {
      this.noMic = true;
    }

    if (videoTrack) {
      this.tracks.push(videoTrack);
      this.noVideo = false;
    } else {
      this.noVideo = true;
    }

    this.dataTrack = {
      local: new Video.LocalDataTrack({}),
      enabled: false,
    };

    this.tracks.push(this.dataTrack.local);

    this.audioMuted = false;
    this.micMuted = false;
    this.videoMuted = false;

    const room: any = await Video.connect(this.token, {
      name,
      tracks: this.tracks,
      // networkQuality: {
      //   local: 1,
      //   remote: 2,
      // },

      // preferredAudioCodecs: ['isac'],
      // audio: {

      // }
      preferredVideoCodecs: ['H264'],
      // preferredAudioCodecs: [ 'opus' ],

      // logLevel: 'debug',
      // networkQuality: {
      //   // remote: 1,
      // }
    }).catch((e) => {
      this.connecting = false;
      throw e;
    });
```

Let me know if you need anything else...

LOGS
```
core.js:40854 Angular is running in the development mode. Call enableProdMode() to enable the production mode.
client:53 [WDS] Live Reloading enabled.
log.js:139 2020-09-16 00:46:38.408Z | INFO in [createLocalTracks #1]: Call to getUserMedia successful; got MediaStreamTracks: [0: MediaStreamTrack, 1: MediaStreamTrack]
log.js:139 2020-09-16 00:46:38.409Z | DEBUG in [LocalAudioTrack #1: undefined]: Initializing
log.js:139 2020-09-16 00:46:38.492Z | DEBUG in [LocalVideoTrack #2: undefined]: Initializing
log.js:139 2020-09-16 00:46:38.683Z | DEBUG in [LocalVideoTrack #2: 483185d7-bac4-4044-8ea4-1bbe10cea0f6]: Attempting to attach to element: <video autoplay playsinline>​</video>​
log.js:139 2020-09-16 00:46:38.809Z | DEBUG in [LocalAudioTrack #1: d2c431e3-4876-43b8-a7e1-d88a9d99be13]: Started
log.js:139 2020-09-16 00:46:38.997Z | DEBUG in [LocalVideoTrack #2: 483185d7-bac4-4044-8ea4-1bbe10cea0f6]: Dimensions: {width: 480, height: 640}
log.js:139 2020-09-16 00:46:38.997Z | DEBUG in [LocalVideoTrack #2: 483185d7-bac4-4044-8ea4-1bbe10cea0f6]: Started
```

**Software versions:**

 - [ ] Browser(s): iOS Safari
 - [ ] Operating System: iPad iOS 13.7 / iPad 6th gen
 - [ ] twilio-video.js: 2.7.2
 - [ ] Third-party libraries (e.g., Angular, React, etc.): Angular

**Comment by manjeshbhargav** [comment]
Hi @KleptoKat ,

Sorry for the delayed response. Can you give me access to your application so I can do some debugging?

Thanks,

Manjesh

**Comment by manjeshbhargav** [comment]
Hi @KleptoKat ,

I'm closing the issue due to lack of activity. Please feel free to reopen when you can give me access to your app so that I can do some debugging.

Thanks,

Manjesh

**Comment by KleptoKat** [comment]
> Hi @KleptoKat ,
> 
> I'm closing the issue due to lack of activity. Please feel free to reopen when you can give me access to your app so that I can do some debugging.
> 
> Thanks,
> 
> Manjesh

Hey @manjeshbhargav ,

Apologies for the delay here. Can I have your email to send you access to the repo? It's hosted on gitlab. Additionally, I can give you credentials to our staging application.

J
